src/GraphQL/API/Enum.hs

```diff
@@ -50,6 +56,14 @@ instance forall conName f p b.
   genericEnumToValue (L1 _) =
     case nameFromSymbol @conName of
       Right name -> name
+      -- XXX: This is impossible to catch during validation, because we cannot
```

Comment:
That still relies on someone remember to run that code first before calling this code, right?